site/src/components/popover.md

```diff
@@ -50,57 +61,118 @@ You can apply almost any classes on the `summary` element to achieve the look yo
+		laboris nisi ut aliquip ex ea commodo consequat.
+	</div>
 </details>
+
+<div class="display-flex justify-content-center">
+	<details class="popover margin-xxs">
+		<summary class="button">Popover</summary>
+		<div class="popover-content">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat.
+		</div>
+	</details>
+</div>
+
+<div class="display-flex justify-content-flex-end">
+	<details class="popover margin-xxs">
+		<summary class="button">Popover</summary>
+		<div class="popover-content">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat.
+		</div>
+	</details>
+</div>
 ```
 
+## Fixed positioning
+
+You can override the dynamic positioning behavior with the following classes.
+
 ### Vertical alignment
 
-The popover can also show up above the button.
+Use the `popover-top` class to force the popover to appear above the trigger element, regardless of available space.
 
 ```html
-<details class="popover popover-top margin-xxs">
-	<summary class="button">Popover on top</summary>
-	<div class="popover-content">Popover content.</div>
-</details>
+<div class="display-flex justify-content-center">
+	<details class="popover popover-top margin-xxs">
+		<summary class="button">Popover top</summary>
+		<div class="popover-content">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat.
+		</div>
+	</details>
+</div>
+```
+
+Use the `popover-bottom` class to force the popover to appear below the trigger element.
+
+```html
+<div class="display-flex justify-content-center">
+	<details class="popover popover-bottom margin-xxs">
+		<summary class="button">Popover bottom</summary>
+		<div class="popover-content">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat.
+		</div>
+	</details>
+</div>
 ```
 
-Vertical and horizontal alignment can be used together.
+### Horizontal alignment
+
+By default, the popover content is centered with the popover button. You can use the `popover-left` or `popover-right` classes on the `.popover-content` element to align the popover content with the left or right edge of the button.
 
 ```html
-<details class="popover popover-top margin-xxs">
+<details class="popover popover-left popover-caret margin-xxs">
 	<summary class="button">Popover aligned to the left</summary>
```

Comment:
Super nit: can you please update summary's text on these examples, so alignment demos are more obvious?

Just the suggestions, whatever else short will work :)

`Popover aligned to the left` -> `Left-aligned`
`Popover aligned to the right` -> `Right-aligned`

<img width="1504" height="710" alt="image" src="https://github.com/user-attachments/assets/ed95608e-0143-44ef-b074-44ead4ba21c3" />